k8s/prod/common/bsp/bulk-scan-orchestrator.yaml

```diff
@@ -31,6 +31,12 @@ spec:
         TRANSFORMATION_URL_DIVORCE: ""
         DELETE_ENVELOPES_DLQ_MESSAGES_CRON: "0 30 0/2 * * *"
         DELETE_ENVELOPES_DLQ_MESSAGES_TTL: "PT72H"
+        TRANSFORMATION_URL_BULKSCAN: "http://bulk-scan-sample-app-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+        TRANSFORMATION_URL_PROBATE: "http://probate-back-office-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+        TRANSFORMATION_URL_FINREM: "http://finrem-cos-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+        TRANSFORMATION_URL_SSCS: "http://sscs-bulk-scan-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+        UPDATE_URL_BULKSCAN: "http://bulk-scan-sample-app-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+        UPDATE_URL_PROBATE: "http://probate-back-office-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
```

Comment:
I mean dynamic replacement won't work here

Reply:
my bad, didn't notice that. Udpated the urls now

Reply:
Reverted the changes, as discussed on the https://github.com/hmcts/bulk-scan-orchestrator/pull/1072